adagrams/game.py

```diff
@@ -1,11 +1,155 @@
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
 def draw_letters():
-    pass
+
+    final_string = []
+    letters_frequency = []
+    letters_list = []
+    
+    for letters in LETTER_POOL:
+        letters_frequency.append(LETTER_POOL[letters])
+    
+
+    for letter in LETTER_POOL:
+        letters_list.append(letter)
+
+    while len(final_string) < 10:
+        letter_index = random.randint(0, len(LETTER_POOL) -1)
+
+        if letters_frequency[letter_index] > 0:
+            letters_frequency[letter_index] -= 1
+            final_string.append(letters_list[letter_index])
+        
+    return final_string
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    available_letters = []
+
+
+    for letter_item in letter_bank:
+        available_letters.append(letter_item)
```

Comment:
Looks like you wanted to make a copy of `letter_bank`. You could get the same functionality with `list(letter_bank)` to make a copy as well. Removing the letters from this list won't affect the original `letter_bank` since the elements are only being removed from the copy.

Reply:
Do I need to make this change, or take it as a suggestion?